### ISSUE TITLE
Which excluded volume implementation should be used?

### ISSUE THREAD

**Issue by hrabbel** [LeMonADE-project/LeMonADE#5]
There is FeatureExcludedVolume in the source, but nobody seems to be using it in our group. Rather, FeatureExcludedVolumeSc is used as the de-facto standard, but it comes from my personal projects. So the latter could be incorporated into the standard repo. Some related arguments:
- FeatureExcludedVolume can be used with SC and BCC bfm. This is not the case for FeatureExcludedVolumeSc
- Interactions via nearest neighbour contacts are implemented for FeatureExcludedVolumeSc, and not for FeatureExcludedVolume. A test-implementation for interactions in combination with FeatureExcludedVolume was very slow due to the many lattice sites that had to be checked.
- Does anyone use BCC bfm at all??

**Comment by Bondoki** [comment]
Yes, we discussed already this issue. Most of my programs/projects using the more general feature FeatureExcludedVolume because it's up to now our standard feature. Most likely I haven't any (explicit) interaction model except for the GPU implementation of lipids and dendrimer/solvent. But they use a different type of ex vol due to CUDA intrinsics. There's no problem to introduce a specialized variant as FeatureExcludedVolumeSc and maybe also FeatureExcludedVolumeBcc. The latter I used for simulation featuring the BccBFM, which is indeed done and used by FeatureExcludedVolume. Maybe we write an FeatureExcludedVolumeGeneral (for scBFM and bccBFM) and one specialized version only for scBFM as you already did. I still prefer the support of the bccBFM.
